### ISSUE TITLE
Map module link

### ISSUE THREAD

**Issue by bpayst** [modolabs/Kurogo-Mobile-Web#61]
The KML file linked causes an incorrect URL to get passed to the browser. the URL is missing the hostname. Test site is here:

http://ec2-184-73-150-55.compute-1.amazonaws.com/map/index?group=Ops&listview=1&_b=%258B%258E%2505%2500

click on Hyatt TNC to see behavior.

feeds-Ops.ini that contains this:

[0]
TITLE = "Hyatt"
SUBTITLE = "TNC"
BASE_URL = DATA_DIR"/Hyatt.kml"
CONTROLLER_CLASS = "MapDataModel"
STATIC_MAP_CLASS = "GoogleStaticMap"
JS_MAP_CLASS = "GoogleJSMap"
SEARCHABLE = 0
DEFAULT_ZOOM_LEVEL = 15

[1]
TITLE = "KMZ"
SUBTITLE = ""
BASE_URL = DATA_DIR"/Hyatt.kmz"
CONTROLLER_CLASS = "MapDataModel"
STATIC_MAP_CLASS = "GoogleStaticMap"
JS_MAP_CLASS = "GoogleJSMap"
SEARCHABLE = 0
DEFAULT_ZOOM_LEVEL = 15

**Comment by bpayst** [comment]
update, demo is here: http://ops.tnc.org/map/

click the list view button and you get the same error.

or direct link

http://ops.tnc.org/map/index?group=Ops&listview=1&_b=%258B%258E%2505%2500

looks like it is happening when the map module processes the URL it is receiving

The underlying KML is successfully displayed by Kurogo if you add the host name:

http://ops.tnc.org/map/campus?feed=12e1bd142d&group=Ops&featureindex=0&category=12e1bd142d&_b=%258B%25AEV%252AQ%25B2R%25F2M%252CP%25D2Q%25CAA0%250B%2580%25AC%25CC%25BC%2594%25D4%250A%2B%253B%2511%25C8N%252F%25CA%252F-%25B0%25F5%252F%2528V%25CB%25C9%252C.%2529%25CBL-%25B75T%25AA%258D%2505%2500

**Comment by bpayst** [comment]
this appears to be caused by having only one point in the KML file. Adding an additional point allows the file to be parsed correctly.

**Comment by akinspe** [comment]
Does this issue still exist?

**Comment by bpayst** [comment]
Yes, if you add multiple points to the KML it functions, but with one it returns an error

**Comment by akinspe** [comment]
Could you please post an updated link to a KML file that does not function?
